05_testing_and_ci/automation_exercise.md

```diff
@@ -1,92 +1,35 @@
-# Automating workflows with GitHub Actions and GitLab CI
+# Automating Workflows with GitHub Actions
 
-In this exercise sheet you create automated workflows and pipelines based on GitHub Actions and GitLab CI. At the end of the exercise you find a section with hints and remarks.
+In this exercise, we create automated workflows and pipelines based on GitHub Actions. We are again working with our beloved diffusion Python code.
 
-Deadline: **Thursday, January 27th, 2022, 09:00**
+Deadline: **Thursday, February 2nd, 2023, 09:00**
 
-## Task descriptions
-
-The CI workflow consists of the three steps:
-
-1. Check the code style of the Python files in the repository using [`black`](https://github.com/psf/black). This check should fail if `black` finds any file to reformat.
-2. Run tests while also collecting coverage information via `pytest` and the `coverage` package. The coverage information (stored in the file `.coverage`) has to be stored and handed over to the next step.
-3. Create a coverage report (`coverage report`) in the terminal and afterwards also convert the coverage information into XML format (`coverage xml`). For this you have to reuse the coverage information (i.e. `.coverage`) from the previous step. The resulting file `coverage.xml` should be kept for 14 days.
-    - Running `coverage report` allows us to inspect the coverage directly in the workflow's output. Saving the XML file allows us to analyze the coverage in more detail with other tools if needed.
-
-Once the workflow runs successfully, add a badge for your pipeline/workflow to the `README.md` (GitHub) or repository (GitLab).
-
-## GitHub Actions
+## Preparations
 
-In this subtask you should implement the workflow as GitHub Action.
+Import the [automation exercise repository](https://github.com/Simulation-Software-Engineering/automation-exercise) in your own account/namespace on GitHub and name it `automation-exercise` again. **Note**: We cannot work with forks here because GitHub Actions may not work in pull requests without explicit approval of the owner of the target repository
```

Comment:
The repo you have tagged here has issues which have submissions from last year. Do we plan to close the issues before tomorrow?